build/configs/common_download.sh

```diff
@@ -0,0 +1,390 @@
+#!/usr/bin/env bash
```

Comment:
Could you please separate out commits as below:
1) Add common download script <detailed description>
2) Update rtl8721csm download script & defconfigs as per common download script
3) Update imxrt1020 download script & defconfigs as per common download script

Reply:
Okay, I will update the commits accordingly.

Reply:
Done.